tests/Api/ApiResponseTraitTest.php

```
@@ -0,0 +1,742 @@
+				// In tests, we can't actually send headers, so just return the JSON
+				return json_encode($problem);
+			}
+
+			public function testProblemResponse(
+				ProblemType|string $type,
+				?string $title = null,
+				?int $status = null,
+				?string $detail = null,
+				?string $instance = null,
+				array $extensions = [],
+				array $headers = []
```

PHPMD (2.15.0)

Avoid unused parameters such as '$headers'. (undefined)

(UnusedFormalParameter)